ucp/_libs/ucp_py.pyx

```diff
@@ -231,8 +232,48 @@ cdef class Endpoint:
     def __cinit__(self):
         return
 
-    def connect(self, ip, port):
-        self.ep = ucp_py_get_ep(ip, port)
+    @property
+    def local_address(self):
+        return [self.local_ip(), self.local_port()]
```

Comment:
Probably more appropriate to return a tuple. (https://nedbatchelder.com/blog/201608/lists_vs_tuples.html)

Reply:
Or perhaps even a string joining the two together `'{}:{}'.format(self.local_ip, self.local_port)`